### PR TITLE
Added avatars to userlist and userAutoComplete

### LINKED ISSUE #8869: Show user avatar in User Autocomplete and User list page
**Is your feature request related to a problem? Please describe.**
Currently the [`UserAutocompleteFormField`](https://github.com/ohcnetwork/care_fe/blob/develop/src/Components/Common/UserAutocompleteFormField.tsx#L82) and the Users list page does not show the avatar of the user.

**Describe the solution you'd like**
- [ ] Show the avatar of the user in each of the options for [`UserAutocompleteFormField`](https://github.com/ohcnetwork/care_fe/blob/develop/src/Components/Common/UserAutocompleteFormField.tsx#L82)
- [x] Show the avatar of the users in the Users list page ([here](https://github.com/ohcnetwork/care_fe/blob/develop/src/Components/Users/ManageUsers.tsx#L192))

**Additional context**
Image

Image

### PR DESCRIPTION
**Pull request by noufalrahim**
## Proposed Changes

- Fixes #8869 
-  Added Avatar in userAutoComplete and userList
Screenshot from 2024-10-23 19-55-49
Screenshot from 2024-10-23 19-56-35

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
lgtm; but could you discard changes made to the lockfile?

**Comment by noufalrahim** [comment]
Done!

**Comment by noufalrahim** [comment]
Why is Cypress test failing? Is there anything that needs to be done from my end?

**Review by rithviknishad — APPROVED** [review]

**Comment by rithviknishad** [comment]
seems to be a flaky test; fixed in https://github.com/ohcnetwork/care_fe/pull/8891

cc: @nihal467

File: package-lock.json [hunk 1]
```diff
@@ -21311,4 +21311,4 @@
       }
     }
   }
-}
+}
```
Comment (bodhish) [inline package-lock.json:21314]: Unrelated change, do we need this 🤔

File: src/components/Common/UserAutocompleteFormField.tsx [hunk 2]
```diff
@@ -80,8 +80,11 @@ export default function UserAutocomplete(props: UserSearchProps) {
         )}
         optionLabel={formatName}
         optionIcon={userOnlineDot}
+        optionImage={(option: UserBareMinimum) =>
+          option.read_profile_picture_url || ""
```
Comment (bodhish) [inline src/components/Common/UserAutocompleteFormField.tsx:84]: why should we cast it to an empty string 🤔, we should be able to deal with an option right 🤔

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/components/Common/UserAutocompleteFormField.tsx (hunk 2 above)
Comment (noufalrahim) [reply src/components/Common/UserAutocompleteFormField.tsx:84]: Screenshot from 2024-10-24 11-31-36
There is an error

**Comment by noufalrahim** [comment]
Opened another PR

File: src/components/Common/UserAutocompleteFormField.tsx (hunk 2 above)
Comment (rithviknishad) [reply src/components/Common/UserAutocompleteFormField.tsx:84]: you can make the `optionImage` accept undefined right?

**Comment by rithviknishad** [comment]
Why close and re-create PR?

**Comment by noufalrahim** [comment]
Merge conflict was there..so I undo everything, created a new branch and committed.

**Comment by rithviknishad** [comment]
https://docs.ohc.network/docs/contributing/#how-to-resolve-merge-conflicts
